app/code/core/Mage/Captcha/etc/config.xml

```diff
@@ -157,6 +157,15 @@
                 </captcha>
             </updates>
         </layout>
+        <translate>
+            <modules>
+                <captcha>
```

Comment:
shouldn't this be `<Mage_Captcha>` ?

Reply:
Yes, you're right. 

Did not noticed this